drivers/tests/UNITTESTS/AnalogIn/test_analogin.cpp

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2021 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* This empty test file is present in order to generate coverage data for the
+ * corresponding source file. This file allows an executable CMake target to be
+ * created, which builds the corresponding source file and produces a .gcno
+ * file to be used by a coverage tool.
+ *
+ * Replace this main function with Google Test tests, as described in the
+ * Mbed OS documentation:
+ *
+ *      https://os.mbed.com/docs/mbed-os/latest/debug-test/unit-testing.html
+ *
+ */
+int main() {
```

Comment:
`{` new line (I assume the astyle run in github action will catch this

Reply:
Oops! Thanks for spotting, will amend

Reply:
`^drivers/tests/UNITTESTS` is in codeignore. I wonder why we add almost all unittests into codeignore , I've missed this being added.

Reply:
We should add a commit to remove that folder from codeignore. We ought to be checking coding style in our unit tests.

Reply:
When we reworked the directory structure and moved files around last year, there were a number of style non-compliances in unit tests, and the decision was to ignore them and come back to this in the future. But fixing them all at once is super easy - just run astyle on all of them, and we should've done this back then.

Reply:
I have ran astyle on everything and removed the directory from the codecheckignore.

Reply:
nice!